applications/asset_tracker_v2/src/ext_sensors/Kconfig

```diff
@@ -9,6 +9,14 @@ menuconfig EXTERNAL_SENSORS
 
 if EXTERNAL_SENSORS
 
+config EXTERNAL_SENSORS_SET_ACCELEROMETER_TRIGGER_ON_INIT
+	bool "Set accelerometer trigger on initialization"
+	help
```

Comment:
I am sorry, but this text does not explain in a way that I understand it, what happens. 

> If this option is set the internal callback for accelerometer triggers is set upon initialization of the library.

What happens when the internal callback for accelerometer triggers is set upon initialization of the library?

> If this option is disabled accelerometer threshold triggers
> must be manually enabled via the ext_sensors_accelerometer_trigger_callback_set() function.

Why must I manually enable them? How do I do that?

Reply:
Ill try to rephrase this in more practical terms. What this actually means for the application.

Reply:
I updated the description to contain a little more high-level text. Manually enabling usually means that the functionality is not automatically enabled during initialization. So a certain API call is needed to enable the functionality.

Reply:
Well, I guess in this context "manually" means, programmatically.